public/search.html

```diff
@@ -0,0 +1,275 @@
+            </li>
+            <li id="carousel__slide4"
+                tabindex="0"
+                class="carousel__slide">
+              <div class="carousel__snapper"></div>
+              <a href="#carousel__slide3"
+                 class="carousel__prev">Go to previous slide</a>
+              <a href="#carousel__slide1"
+                 class="carousel__next">Go to first slide</a>
+            </li>
+          </ol>
+          <aside class="carousel__navigation">
+            <ol class="carousel__navigation-list">
+              <li class="carousel__navigation-item">
+                <a href="#carousel__slide1"
+                   class="carousel__navigation-button">Go to slide 1</a>
+              </li>
+              <li class="carousel__navigation-item">
+                <a href="#carousel__slide2"
+                   class="carousel__navigation-button">Go to slide 2</a>
+              </li>
+              <li class="carousel__navigation-item">
+                <a href="#carousel__slide3"
+                   class="carousel__navigation-button">Go to slide 3</a>
+              </li>
+              <li class="carousel__navigation-item">
+                <a href="#carousel__slide4"
+                   class="carousel__navigation-button">Go to slide 4</a>
+              </li>
+            </ol>
+          </aside>
+        </section>
+      </section>   
+    -->
+    </main>
+
+
+    <footer class="footer">
+      <div class="track-info">
+        <img width="56" height="56" src="%PUBLIC_URL%/images/7.jpg" alt="IVOXYGEN - room">
+        <div class="track-info-head">
+          <a href="" class="plaing-track-name">room</a>
+          <a href="" class="plaing-track-autor hover-white">IVOXYGEN</a>
+        </div>
+        <button class="plaing-track-like btn-hover-white">
+          <svg viewBox="0 0 16 16" width="16" height="16">
+            <path d="M1.69 2A4.582 4.582 0 018 2.023 4.583 4.583 0 0111.88.817h.002a4.618 4.618 0 013.782 3.65v.003a4.543 4.543 0 01-1.011 3.84L9.35 14.629a1.765 1.765 0 01-2.093.464 1.762 1.762 0 01-.605-.463L1.348 8.309A4.582 4.582 0 011.689 2zm3.158.252A3.082 3.082 0 002.49 7.337l.005.005L7.8 13.664a.264.264 0 00.311.069.262.262 0 00.09-.069l5.312-6.33a3.043 3.043 0 00.68-2.573 3.118 3.118 0 00-2.551-2.463 3.079 3.079 0 00-2.612.816l-.007.007a1.501 1.501 0 01-2.045 0l-.009-.008a3.082 3.082 0 00-2.121-.861z"></path>
+          </svg>
+        </button>
+        <button class="plaing-track-toggle btn-hover-white" >
+          <svg width="16" height="16">
+            <path  d="M1 3v9h14V3H1zm0-1h14a1 1 0 0 1 1 1v10a1 1 0 0 1-1 1H1a1 1 0 0 1-1-1V3a1 1 0 0 1 1-1z"></path>
+            <path  d="M10 8h4v3h-4z"></path>
+          </svg>
+        </button>
+      </div>
+
+      <div class="track-panel">
+        <div class="control-btns-container">
+          <div class="control-btns">
+            <button class="random-tracks control-btn btn-hover-white">
```

Comment:
По умолчанию, кнопка имеет type=submit.
Тут стоит сразу указать корректный тип
```suggestion
            <button type="button" class="random-tracks control-btn btn-hover-white">
```

Это относится ко всем местам

Reply:
исправил